dlp/snippets/deid_test.py

```diff
@@ -0,0 +1,291 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import shutil
+import tempfile
+
+import google.cloud.dlp_v2
+import pytest
+
+import deid
+
+HARMFUL_STRING = "My SSN is 372819127"
+HARMLESS_STRING = "My favorite color is blue"
+GCLOUD_PROJECT = os.getenv("GOOGLE_CLOUD_PROJECT")
+UNWRAPPED_KEY = "YWJjZGVmZ2hpamtsbW5vcA=="
+WRAPPED_KEY = (
+    "CiQAz0hX4+go8fJwn80Fr8pVImwx+tmZdqU7JL+7TN/S5JxBU9gSSQDhFHpFVy"
+    "uzJps0YH9ls480mU+JLG7jI/0lL04i6XJRWqmI6gUSZRUtECYcLH5gXK4SXHlL"
+    "rotx7Chxz/4z7SIpXFOBY61z0/U="
+)
+KEY_NAME = (
+    "projects/python-docs-samples-tests/locations/global/keyRings/"
```

Comment:
Since you're running tests in version-specific projects, please create this key in every project and use it that way. We don't want inter-project dependencies. They make management much harder.

Once you make the change let me know and I'll sort out the permissions in there are still problems.

Reply:
Sure, makes sense. 
I created the keyring and key in Python3.7 
Didn't have the permission to create in other projects.

Reply:
I created the keyrings and keys in every test project.

Reply:
Please remember to change the project part of the key name to use the project-specific keys :)

Reply:
Thanks Maciej for creating the keys.
I already modified the project part of the key. The errors are not due it. I'm looking into it now.